add: lavaland rework 1.5

NightDawnFox opened the pull request:
<!-- Пишите **НИЖЕ** заголовков и **ВЫШЕ** комментариев, иначе ваш текст может не отобразиться. -->
<!-- В Contributing.MD вы можете найти некоторые рекомендации к оформлению пулл-реквеста. -->

## Описание
Увеличение контента для шахты, направленное в основном на фауну и различные QoL улучшения.

### **Краткий список изменений:**
###### все изменения указаны в произвольном порядке. 

- Полная переработка мегафауны "Легион" - изменение его поведения, атак, принципа боя с ним. Добавление легиону уникального трофея на крашер.

- Переработка гемов. Уменьшение цены за сканирование и продажу гемов. Добавление практически всем гемам новых особенностей и взаимодействия. Добавление бижутерии из гемов.

- Переработка голиафов. Теперь голиафов можно "приручить", если в достаточной мере покормить их мясом или кактусами. Теперь голиафы вблизи создают защитную "стену" из тентаклей, защищаясь от людей с крашером. Теперь голиафы с шансом в 50% вместо своей стандартной дальней атаки могут атаковать чарджем, быстро сближаясь с противником. Теперь часть проджектайлов с шансом в 30% (50% для древних) может отразиться от шкуры голиафа и полететь в разные стороны или отразиться обратно в игрока.

- Изменение проджектайлов у наблюдателей. Все лучи наблюдателей были замедлены, уворачиваться от них стало проще. Теперь луч обычного наблюдателя, вместо скучной заморозки накладывает на игрока смятение, замедление и роняет на секунду, а так же наносят 10 урона ожогами. У замораживающего наблюдателя длительность "нахождения в заморозке" увеличена с 0.8 до 1.5 секунд. У поджигающего наблюдателя значительно увеличена температура, длительность горения, а так же, в случае, если у цели более 50 урона ожогами, при попадании нового луча на цель накладывается секундный стан. Повышено здоровье некоторых наблюдателей.

- Переработка последствий встречи с легионами. Теперь, в случае если ваше тело достанут из легиона, вместо того, чтобы иметь 1000 урона и переломанные кости, ваше тело будет в идеальном состоянии, требующим исключительно дефибриллятора. Таймер смерти сбрасывается, как и повреждения внутренних органов, восстанавливаются конечности (как при вшитом ядре). Однако за это нужно платить: отныне игрок может заразиться опухолью легиона. Способа получения опухоли два - заражение через атаку черепка и последствия "спасения" из легиона. Принцип действия опухоли, шансы заражения, последствия и лечение в соответствующем пункте ниже. Повышено здоровье легионов.

-  Добавлен новый подтип вашего любимого монстра - Морозный паук. В отличие от обычного паука, при укусе вводит в кровь морозное масло. Разозлённый, так же начинает кусать веномом. В остальном идентичен собрату. 

- Отныне чазмы не гибают игрока навсегда (если он не робот. Роботов гибают.), а сохраняют их в себе. Отныне шахтеры могут спасти своих менее удачных, попавших в чазм, коллег с помощью очень дорогой и специализированной гранаты, телепортирующих их к любому маяку на Z-уровне станции или.. с помощью удочки?

- Отныне генокрады, попавшие в чазм, могут оттуда выбраться, если оживут, находясь в чазме. Тогда их выбросит к краю чазма и они смогут продолжить игру.

- Все шахтерские инструменты для копки были улучшены. В целом, скорость всех инструментов была увеличена на 0.1-0.2 секунды. Исключение - Джекхаммер. Отныне он ломает породу мгновенно (для укрепленной породы все еще требуется от двух до трёх ударов). Стены при этом он продолжает ломать с обычной скоростью.

- Незначительные багфиксы от прошлого ПРа: луки теперь можно хранить на спине, в колчанах держится по 21 стрела (кратно трём).  Теперь шаман пеплоходцев станит с вероятностью в 25%, вместо 100%, как раньше. Теперь шанс на получение с дрейка улучшенной драконьей крови (превращает в драконида) равен 1 к 5, а не 1 к 16, как было раньше. Повышена защита у перчаток голиафа. Теперь накидка первопроходца может хранить в себе то же самое, что хранит в себе обычная накидка голиафа. Значительно порезан лут с пауков. Теперь базальтовая катана наносит нормальный повышенный урон фауне.

- Переработка колосса. Отныне колосс говорит только "колоссальным" шрифтом. Отныне между каждой атакой колосса есть значительная задержка, а при тяжелых атаках (спиральная атака) меняется анимация колосса. В случае, если у колосса осталось менее 10% здоровья, он использует одноразовую особую, ужасающе разрушительную атаку, способную убить неподготовленного шахтера. В целом - колосс стал значительно проще для убийства с помощью крашера и просто проще для ЕКА. В обмен на это - глас бога может выпасть только при убийстве крашером.

- Добавление шахтерских раций! Будучи обычными рациями, они, помимо прочего, способны предсказывать погоду, оповещая игрока визуально и словами о том, когда будет буря. Оповещения идут за 2 минуты и за 30 секунд до начала таймера погоды. Спрайты сделал наш глав.спрайтер.

- Изменение шпилей! Отныне шпили можно помечать сканером руды, после чего те начнут показываться на GPS, показывая тип шпиля (легионы - LG, наблюдатели - WT, голиафы - GL) и уникальную цифру. При уничтожении шпиля, гпс пропадает. Добавлено два новых шпиля - случайный, который спавнит все типы обычной фауны раз в 25 секунд (обычный шпиль спавнит фауну раз в 30 секунд), а так же способен спавнить до 5 животных. И второй - "повышенной опасности", спавнящий еще больше фауны, включая уникальные типы, спавнит фауну раз в 15 секунд и держит до 7 фауны вокруг себя. Очень редкий
Здоровье всех шпилей было повышено. Из шпиля с наблюдателями теперь могут вылезать морозные и поджигающие наблюдатели. Уменьшена частота появления легионов из шпиля с ними.

- Добавление новой флоры с шиптеста - огненного цветка. Особенности - светится в темноте, цветки сами по себе имеют ген биолюминесценции и светятся. Можно надеть в слот головы, тогда будет красивый цветок на голове. Да...

- Порт режима "HRD-MDE" для всей мегафауны с оффов парадиза. Подробности ниже.

- Новый предмет для апплинка, уникален для шахтера и квартирмейстера: Овверайдер подрывных зарядов. Цена - 4 ТК. Имеет 6 зарядов. При использовании на подрывных зарядов, заметно снижает их мощность и позволяет ставить их на любом тайле, превращая их в более слабый (для слабой взрывчатки) или заметно более сильный (для экспериметальной взрывчатки) аналог С4. Улучшение давления для КА теперь доступно и квартирмейтеру. Подробности зарядов ниже.

- Все ГПС на аванпостах были перемещены в хранилища шахтерских костюмов. В каждый аванпост был добавлен пост ГПС с названием "OUTPOST"

- Была добавлена джаунтерная граната с оффов. При использовании она создает червоточины ко всем маячкам на уровне станции. Полезна, чтобы замедить офицеров при побеге. Так же используется при спасении шахтеров с чазмов. Подробнее ниже.

- Все цены на предметы в шахтерских вендорах были в основном незначительно повышены на 200/500 ОДР. Всё равно очков у шахтеров всегда слишком много.

- В вендоры был добавлен новый предмет - шприц с углем. 

- Полный порт посоха бурь с оффов/ТГ. Подробности ниже.

- Вместимость модуля "Hardness" была понижена до 30%. Поставлено ограничение на один модуль в КА

- Отныне на Z-уровне станции нельзя развернуть шахтерскую капсулу до тех пор, пока она не будет емагнута. В космосе на других Z-уровнях, ну и на лаве открывать капсулы можно.

- Очередное обновление ТГУИ шахты. Теперь карты способны отслеживать, сколько руды было добыто за всю смену.

- Незначительные твики пеплоходцам: Теперь все пеплоходцы, включая шамана, и дракониды имеют новую способность - они могут раз в 40 секунд получить себе "зажигалку", которую они могут использовать для того, чтобы поджигать сигареты или типа того. У шамана пеплоходцев появилась новая способность, позволяющая ему определять примерное местоположение (относительно себя) мировой кузни. Отныне пеплоходцы могут использовать все магмитовые инструменты (необходимо потом обновить вики по этому поводу).

 - Отныне пеплоходцы и люди, использующие облик пеплоходцев, полностью теряют скорость, когда покидают лаваленд. Никаких более генокрадов - соников в облике пеплоходца с быстрыми ногами по станции.

 - Теперь уничтожение шпиля пеплоходцев даёт нормальное случайное ядро аномалии вместо странного сломанного ядра без технологий.

 - Изменены шансы на появление фауны. Теперь наблюдатели на карте появляются с шансов 60/20/20, где 60 - обычный наблюдатель, 20 - замораживающий наблюдатель, 20 - огненный наблюдатель. Теперь древние голиафы появляются с шансом в 10%, а не 1%, как это было раньше.

### **Подробные изменения:**

###### Порядок всё еще произвольный, отмечены наиболее важные моменты.

### Переработка Легиона
Данные изменения - порт с оффов нового поведения легиона.
Основные моменты: 
Теперь Легион более не разделяется при убийстве, вместо этого он имеет здоровье в 2500 единиц. 
Отныне в ближнем бою он наносит столько же урона, сколько и остальная мегафауна.
Теперь его спрайт анимирован. По мере уменьшения здоровья спрайт легиона будет уменьшаться.
Каждый раз, когда в него попадают (ЕКА, любой выстрел, удар крашером) он будет призывать маленькие черепки.
Новые атаки легиона:
Чардж.
Чардж теперь стал более опасен из-за повышенной скорости и урона. Теперь не получится просто убегать от Легиона
Лазер.
Абсолютно новая атака - легион создает лазерный след длинной в целый тайл, после чего игроку дается от 1.25 до 0.25 секунд для реакции и отхода от линии огня, в зависимости от здоровья Легиона. Лазер наносит от 50 до 70 урона в зависимости от здоровья. В случае, если идти по линии лазера, игрок будет получать урон за каждый тайл, по аналогии с хождением по лаве. Лазер превращает в пыль трупы тех, кто не принадлежит к фракции лаваленда.
Призыв.
Легион во время боя начинает призывать обычных легионов, которые могут быть как источником проблем - большего количества отвлекающих черепков, так и источников ядер во время боя. С небольшим шансом Легион так же может призвать больших легионов - медлительных, но бьющих в ближнем бою.
Теперь из легиона выпадает трофей на крашер, позволяющий после детонации использовать дэш по нажатии хелп интента в любую сторону

https://github.com/ss220-space/Paradise/assets/116907157/022ab4dc-2186-42b2-a35d-848a5dbe0f9b


### Режим "HRD-MDE" для мегафауны
Данные изменения - порт с оффов гигантской работы товарища "Qwertytoforty" по добавлению опционального, усложнённого режима для всей мегафауны в игре. Особенность режима в том, что игрок, решивший начать охоту за мегафауной, не получает никаких особых наград за исключением медалей, показывающих его крутость. 

Для того, чтобы приступить к новому режиму, предварительно необходимо приобрести специальный набор в вендоре за 2500 поинтов. Внутри будет находиться коробочка с медалями, куда их можно складывать, пояснительная записка, объясняющая принципы работы данного режима и лорное объяснение, а так же 7 гранат, для семи типов мегафауны в игре.

Активация режима проста - вы просто должны бросить гранату к мегафауне и дождаться ее активации, а после - убить мегафауну ради медали.

Вся мегафауна получила значительное усложнение и несколько новых приёмов, расскажу о них вкратце
У кровавого шахтера был повышен урон его атак, понижено время для его дэша, повышен урон от КА, увеличена частота смены режимов его оружия, увеличена частота атак
Иерофант же в ближнем бою, помимо стандартного увеличения частоты атак, начинает отталкивать игрока от себя, не давая людям с крашером получать бесконечные бэкстебы, а так же начинает бить в ближнем бою.
Легион в сложном режиме делится надвое, увеличивая частоту своих атак.
Дрейк начинает атаковать своими атаками второй фазы в первой фазе. Увеличиваются размеры и длительность лавовых бассейнов, размер арены вырастает до 5x5
Колосс увеличивает частоту своих атак, а так же получает умение использовать "финальную атаку" несколько раз
Ветус увеличивает у себя пробитие брони от атак, обучается стрелять из лазера а так же начинает использовать особые атаки других "вариаций" себя, заметно усложняя бой.
Баблгам же... делает забавное. Found out IC.

Как уже было сказано, особой награды за сложный режим, за исключение медалей нет. Убийство всей мегафауны в сложном режиме дарует игроку особую медаль.

### Легион. Опухоль Легиона
Всё здоровье у легионов было повышено. 100 у обычного, 80 у карлика.
После смерти легиона из него отваливается три агрессивных черепка, атакующие вас.
В случае, если легион вас убьет и превратит в одного из них, ваше освобождение приведёт к полному восстановлению тела, как это происходит при прожатии вшитого ядра. Таймер смерти сбрасывается и игроку, что вас освободил, достаточно лишь один раз ударить дефибриллятором, чтобы запустить сердце, но есть одно но:
Появилась новая уникальная механика: "заражение"
С шансом в 1% каждая атака черепка легиона, в случае если у вас нет хардсъюта, может заразить вас новой неприятностью - опухолью легиона.
Так же, с шансом в 75% вы получите заражение опухолью легиона после того, как вас "освободят" из него.
У заражения опухолью есть 5 стадий, каждая из которых длится по 80 секунд. В сумме заражение длится в районе 8 минут, но на тестах люди жили в районе 13-15 минут. Это напрямую связано с тикером и шансами.
А теперь, как работает заражение:
После наступления второй стадии (спустя ~160 секунд) каждый тик с шансом (текущая стадия / 5) на вас подействует эффект ядра, залечивая раны и убирая замедление, однако..
**На второй и третьей стадии** каждый тик, вы можете:
с шансом в 1% получить сообщение "Your chest spasms!"
с шансом в 1% получить сообщение "You feel weak."
с шансом в 1% получить в голову страшные звуки.
с шансом в 2% вас вырвет, и вы получите стан.
**На третьей и четвертой стадии** каждый тик вы можете:
с шансом в 2% получить сообщение "Something flexes under your skin."
с шансом в 2% получить в голову страшные звуки.
с шансом в 3% вас вырвет кровью, и вы получите стан.
При этом, в 50% со рвотой из вас вылезет черепок легиона, который будет жить столько же времени, сколько живет обычный черепок, однако он будет к вам лоялен.
с шансом в 3% вы получите 20 брут урона и сообщение "Your muscles ache."
**Как только вы достигнете пятой стадии**, в чат высветится сообщение "Something is moving under your skin!" и после этого каждый тик с шансом в 10% вы можете обратиться в нового легиона.
Опухоль удаляется операцией в лазарете. Ну и, в случае если вы заразились опухолью и стали легионом, то опухоль из тела исчезнет, и ваше простое убийство и оживление вас спасёт. Однако, есть шанс в 25% на то, что вы повторно заразитесь опухолью.

### Переработка гемов. Бижутерия из гемов.
Цена всех гемов была **значительно понижена**, однако почти каждый ген получил новое, интересное, в целом бесполезное применение. Были значительно повышены шансы на выпадение гемов из фауны. Мало кто вообще знал, что из златожоров и нестандартных наблюдателей вообще падают гемы из-за их крайне низкого шанса.
Новые способы взаимодействия:
**Рупий** (гем златожора)
-Расплавка дает 30 радия заместо 10
-Теперь его можно разломать острым предметом. В таком случае тот, кто это сделал, получит единовременный выплеск в виде 50 рад.урона, а сам гем начнет излучать радиацию в количестве 10 рада каждые 3 секунды. Зачем это делать? Хороший вопрос.
**озолоченная раковина** (гем огненного наблюдателя). Нельзя получить с наблюдателей из Шпилей.
-Расплавка теперь дает 50 золота заместо 25
-Раз в 15 секунд гем можно использовать в руке. В таком случае он прибавляет 60 градусов к температуре тела, после чего тускнеет. до отката. Зачем?  Это мило.
**Замороженный алмаз** (гем морозного наблюдателя). Нельзя получить с наблюдателей из Шпилей.
-Расплавка теперь дает 30 алмазов вместо 15
-Раз в 15 секунд гем можно использовать в руке. В таком случае он понижает температуру тела на 60 градусов, после чего тускенеет до отката. Зачем? Это мило.
**Стабилизированный барлоксидиум** (гем с кровавого шахтера)
-Теперь имеет в себе 6 материалы и 6 плазму. Можно передать учёным.
**Уплотнённый дилитиум** (гем с иерофанта)
-Теперь, помимо ГПС сигнала, является полностью неуничтожимым. В случае если попадет в чазм, будет оттуда выплюнут.
**Драконий жемчуг** (гем с дрейка)
-Можно расплавить вместо гибтонита на кузне. В таком случае он даст два заряда.
-Используется в создании бижутерии, при надевании наделяет особым эффектом. Подробности ниже.
**Кристалл пустоты** (гем с колосса)
-обзавёлся небольшой анимацией. 
-Каждые 40 секунд способен копировать эффект блюспейс кристалла, блинк - случайную телепортацию в радиусе 6 тайлов.

**Ихор** (гем с бабблгама)
-изменен цвет на более яркий
-В случае, если вы являетесь вампиром, то использование гема в руке позволяет вам единоразово получить 50 активной крови. Имеет уникальное описание, намекающее **исключительно вампиру** о том, что пользователь может получить с него силу: "You can smell human blood coming from the bloodstone."
-В случае, если вы являетесь предателем, то вы можете получить с гема 2 ТК в апплинк, при этом уничтожая гем. Имеет уникальное описание, намекающее **исключительно предателю** о том, что пользователь может зарядить им апплинк: "You have a feeling, that you could you this gem to charge your uplink."

**Блюспейс дата-кристалл** (гем с ветуса)
-я не смог придумать ему ничего крутого
-переплавка теперь приносит 24 бс кристалла вместо 12.
-Чутка уменьшено свечение

***Бижутерия из гемов***
Отныне вы можете скрафтить два новых предмета: Заготовку под амулет и золотые браслеты.
В заготовку по амулет можно вставить любой из существующих гемов и в таком случае он изменит спрайт и вы сможете надеть его на шею или прикрепить к костюму. Надетые на шею предметы сохраняют свечение.
Как оно выглядит, можно глянуть [тут:](https://discord.com/channels/617003227182792704/928706798670721145/1150445083116703814) 
Золотые браслеты аналогично позволяют вставить в себя гемы, однако они ограничены исключительно "простыми гемами" - теми, которые добываются из породы и не представляют из себя ничего ценного. 
В целом амулеты служат исключительно украшательством, за исключением одного конкретного амулета из драконьего жемчуга.
Создание амулета из драконьего жемчуга дарует пользователю новый статус эффект "драконья сила", который пассивно лечит пользователя, когда его здоровье ниже 40%. Пассивное лечение представляет из себя лечение, похожее на эффект "надежда", однако чутка слабее. Не может вытащить из крита из-за своей слабости, неспособно перебить урон от гипоксии в крите, однако при этом поддерживает пользователя приятными словами, когда он ранен, и пишет пугающие  красные сообщения, когда он здоров.

### "Гляньте, кажется я поймал шахтера!"
Отныне попадание карбона в чазм не убивает его. Вместо этого оно сохраняет его во внутреннем хранилище, которое можно даже найти, если у шахтера есть датчики или устройство ГПС. В случае обнаружения неудачного шахтера, вы можете спасти его, используя одно из двух устройста: ужасающе дорогую джаунтерную гранату из вендора, или, что более забавно, выловив неудачного шахтера из чазма с помощью удочки. Джаунтерную гранату необходимо активировать и забросить в тот тайл чазма, в котором находится неудачный шахтер. После этого в чазм ударит молния, как визуальный знак того, что устройство сработало, и шахтер будет случайно перенесен к любому маячку в секторе станции.
Визуальный принцип работы устройства:

https://github.com/ss220-space/Paradise/assets/116907157/9d77d9b1-fe76-454d-86d0-9991973f60b8

Альтернативный способ вытащить шахтера из чазма:

Теперь на каждом аванпосту есть удочка, которую можно использовать для того, чтобы с ее помощью вылавливать шахтеров из чазмов. Ну, тут довольно просто. Активируешь на чазме. Если внутри кто-то есть, вылавливаешь его.

https://github.com/ss220-space/Paradise/assets/116907157/d272361d-cd6e-4561-a1c7-e2801ca9d312

### Шахтерские подрывные заряды. Использование их агентами.
В апплинк предателя был добавлен новый предмет - овверайдер шахтерских подрывных зарядов. Может выпасть в сурплусе, цена - 4 ТК, уникален только для шахтеров и завхоза. Имеет Шесть использований, особое описание для антагониста. Для всех остальных выглядит как обычный сканер для руды.
При использовании на любом подрывном заряде, тратит одно использование, сильно ослабляет мощность взрывчатки, но при этом меняет ее свойство так, чтобы ей можно было пользоваться как обычным пластидом с4.
Мощности взрывчатки:
С4 - 0/0/3
слабая взрывчатка - 0/0/1
обычная взрывчатка - 0/1/4
экспериментальная взрывчатка - 1/2/5
Кто-то скажет, что ты получаешь за 4 ТК до 6 минибомб синдиката, я скажу, что во первых, их можно только устанавливать, а не бросать, что РНД имеет возможность делать более мощную взрывчатку без каких либо затрат, а еще это потребует антагонисту шахтеру зайти и попросить эти самые экспериментальные заряды в РНД, так что в целом цена оправдана. Будет слишком сильно - увеличим цену, уменьшим заряды, скорректируем мощность взрывчатки.

### Джаунтерная граната.
Стоимость - 5000 ОДР. При активации и броске создает множество червоточин в районе броска, каждая из которых ведет к действующему маячку на Z-уровне станции. Высокая стоимость оправдана тем, что сами по себе гранаты будут использоваться для двух вещей - спасать шахтеров (есть более смешная альтернатива) и создавать заслоны в коридорах, мешая офицерам за тобой гнаться.

https://github.com/ss220-space/Paradise/assets/116907157/cc8a08cf-900b-44cb-a4a1-74e97debfeba

Ну и некоторые маячки ведут в космос.

### Новый посох бурь.
Прямой порт с ТГ/оффов. 
Полная смена работы предмета. Теперь он имеет три заряда
Каждый заряд создает зону, в которую через некоторое время ударит молния. Не работает в космосе. На лаваленде урон от молнии выше и при этом цель поджигается. При этом если попасть по кукле, куклу при этом бьет током, игнорируя защиту от электричества.

https://github.com/ss220-space/Paradise/assets/116907157/13022d67-026b-481c-9fb5-1b4f3afc45ba

Дополнительно посох бурь может развеять действующий шторм. Более он не призывает новую бурю в любой локации.


<!-- Опишите, что делает ваш ПР. Документировать каждую деталь не требуется, просто укажите основные изменения. -->

## Ссылка на предложение/Причина создания ПР
https://discord.com/channels/617003227182792704/755125334097133628/1170607673931202630
<!-- Здесь оставьте ссылку на сообщение в #отчеты-по-предложениям, чтобы подтвердить, что ваше предложение одобрено. Либо укажите, почему этот ПР должен пройти без предложки. -->
<!-- Пример ссылки: https://discord.com/channels/617003227182792704/755125334097133628/ID-сообщения -->

NightDawnFox committed:
Исправление багов

Часть файлов лишней оказалась..
Revert "thats all!"

This reverts commit 947524612688087a560cf118f694d7a670a92b85.

NightDawnFox committed:
Revert "that's all!"

This reverts commit 5dc611263ee3e3ec64cdcddfb1b0aba1379083cd.
that's all!
Merge remote-tracking branch 'upstream/master220' into master220

NightDawnFox committed:
1

NightDawnFox committed:
2

NightDawnFox committed:
3

NightDawnFox committed:
Update lavaland_surface_pirateship.dmm
господи блядь как я ненавижу ебаный мапинг убейте меня кто-нибудь
hotfix2
hotfix
new lavaruin - spaceship

NightDawnFox committed:
sprites

NightDawnFox committed:
sprites2

NightDawnFox committed:
yesss

NightDawnFox committed:
mapping 2

NightDawnFox committed:
Закончил с украшениями и гемами.

У гемов теперь уникальные свойства, почти у каждого. Новые украшения, использующие эти самые гемы. В целом, красиво и прикольно.

NightDawnFox committed:
fix

я дебил

NightDawnFox committed:
Новые голиафы, часть 1

Теперь голиафы рычат (через мишку), когда с ними дерешься. Теперь голиафы могут чарджиться на вас и таранить. Теперь вблизи голиафы испускают "защитную стену" из тентаклей. Пока всё, я хочу спать

NightDawnFox committed:
Новые голиафы, часть 2

Теперь голиафов можно "приручить" едой, делая их дружелюбными к тебе. Так же добавляет голиафам шанс на рефлект, который блокирует 30 (50 для древних)% выстрелов, отражая их в стороны и иногда в отправителя. На этом работа с голиафами закончена.

NightDawnFox committed:
Небольшой нерф/бафф наблюдателей

Теперь обычные наблюдатели не замараживают, а роняют куклу, накладывают 3 секунды замедления и конфуза, давая окно в пару миллисекунд перед новым выстрелом, а так же наносят 10 берна за выстрел. Теперь магма и айсвинги спавнятся гораздо чаще

NightDawnFox committed:
Сильный бафф нестандартных вотчеров

NightDawnFox committed:
Доработка Легионов

Теперь тела "игроков", выпавшие из легиона, имеют в себе опухоль, которая таит в себе отложенную смерть (если лазарет решит не клонировать тело, а лечить), превращающая его в легиона. Жууууть.

NightDawnFox committed:
Делаем легионов В РАЗЫ ОПАСНЕЕ

Сейчас легионы представляют из себя мальчиков для битья, не являясь какой-либо угрозой. Теперь же они ставят шахтеров под угрозу выживания. Если им не повезет + шахтерам - они превратятся в новых легионов будучи живыми.
В обмен на это - трупы с легионов падают идеально целыми. Только и дефибай..

NightDawnFox committed:
Увы

NightDawnFox committed:
увы2

унатхи теперь тоже под угрозой. Защищены только люди в ригах.
Умершие от "заражения" имеют шанс в 25% на то, чтобы залутать второй тумор

NightDawnFox committed:
Новый подтип мароу уивера - ледяной

Впрыскивает фростоил. Спрайты с йогов. Смешное

NightDawnFox committed:
Упс.

NightDawnFox committed:
Рыбалка

Теперь миазмы создают в себе хранилище, если туда попадает человек. Теперь на всех аванпостах есть удочка. Теперь шахтеров можно вылавливать из миазм.

NightDawnFox committed:
Баффы ручных шахтерских инструментов

Ну, всё сказано в названии

NightDawnFox committed:
fuck
ой...
Плюс декали, деревянные полы, грязь и мусор.
ой-ёй.
ой.
Update lavaland_surface_seed_vault.dmm

NightDawnFox committed:
Набор багфиксов и балансных правок

1. У перчаток голиафа больше защиты
2. Костюм первопроходца теперь имеет такие же слоты, как и накидка голиафа
3. Усиленная кровь дракона теперь имеет шанс в 1/5, чем 1/16
4. Шаманы пеплоходцев теперь не роняют гарантированно
5. Проджектайлы ватчеров медленные
6. Лук теперь только на спину, добавлены спрайты
7. В колчан теперь лезет 21 стрела, а не 14 (ранее должно было 20, хдд)
Потом нужно фиксить маппинг, ыааа

NightDawnFox committed:
балансные правки 2

Режем дроп у уиверов

NightDawnFox committed:
Мне нельзя доверять баланс.

NightDawnFox committed:
Полировка рыбалки

Теперь генокрады и реджувнутые куклы могут вылезать из чазма

NightDawnFox committed:
"реворк" колосса

Мне в лом расписывать, я просто портировал его атаки и кулдауны с оффов и ТГ. Потом расскажу, че я сделал

NightDawnFox committed:
Шахтерские рации (работают)

Порт с ТГ шахтерских погодных раций, заранее оповещающих шахтеров о приближающейся буре. Код несколько кривой, но работает.. За спрайты спасибо глав.спрайтеру

NightDawnFox committed:
map

NightDawnFox committed:
Кривые лапы

NightDawnFox committed:
да блядь.

NightDawnFox committed:
Пометка шпилей! Новые типы шпилей!

Теперь сканирование тендриллов с помощью сканера руды позволяет их "отмечать" на ГПС и отслеживать их на z-уровне. Добавлено два типа новых шпилей, спавнящие фауну рандомно и чаще. Повышена частота спавна шпилей. Изменен mob_spawn_list Для большего удобства.

NightDawnFox committed:
огненный цветок

Порт нового растения с ТГ/шиптеста - fire blossom. Светится в темноте, при сборе дает светящиеся цветки. Красивые, можно носить на голове

NightDawnFox committed:
posmerge fix
centcomm

NightDawnFox committed:
split-off! screen fog

NightDawnFox committed:
ээээ
ладно, плевать

IconFile::from_bytes failed for "icons/mob/screen_fog.dmi"
46 попыток пофиксить
мда
хмм
Порт режима "HRD-MDE" с оффов

Гигантская работа товарища "Qwertytoforty" по усложению лавы ради медалек теперь будет и у нас (возможно). Нужно проверить и протестить, но у меня уже позднее время.

NightDawnFox committed:
fix 1

NightDawnFox committed:
Казалось бы, одна лишняя скобка

paradise.mdme.dmb - 9772 errors, 1554 warnings (10/22/23 5:21 pm)

NightDawnFox committed:
фиксы маппинга

NightDawnFox committed:
Оверрайдер подрывных зарядов

Превращает безопасные шахтерские подрывные заряды в полноценную взрывчатку. Предмет синдиката. Цифры - позже.

NightDawnFox committed:
Набор портов с оффов и инфляции

1. Добавлен джаунтер-граната, создающая джаунтеры для всех точек на карте и являющаяся альтернативным способом доставать шахтеров с миазм
2. Порт с ТГ посоха легиона
3. В вендоры добавлен шприц с углем
4. Во всех вендорах резко выросла цена на всё.
5. Цена на харднесс стала ниже, однако было добавлено ограничение на один модуль
6. toxix -> toxin в описании инъектора угля
7. На все лавовые карты был добавлен маркер ГПС, ведущий на аванпост
8. Со всех карт были удалены ГПС, они были перемещены в хранилища.

NightDawnFox committed:
Revert: seed vault + shelter capsules tweak

Не хочу ебать себе мозги с прохождением маппинг-предложек..
Более нельзя открывать капсулы на Z-уровне станции, пока ты их не емагнешь.

NightDawnFox committed:
Я даже в реверте умудрился..

NightDawnFox committed:
Твики хардмода мегафауны

Порт с оффов багфиксов, ничего особенного

NightDawnFox committed:
ТГУИ.

Теперь карты отслеживают количество общего количества добытых ресурсов

NightDawnFox committed:
чутка правим рантаймы и переносим карту

ДА.

NightDawnFox committed:
Фиксы багов

NightDawnFox committed:
Проверка для убирания рантаймов

"undefined proc or verb /obj/machinery/porta_turret/syndicate/can inject()

NightDawnFox committed:
Небольшие приколы пеплоходцам ч.1

1. Теперь убийство некрополя пеплов даёт нормальное ядро
2. Теперь у пеплоходцев, шамана и драконида есть встроенная зажигалка. Подробности потом
3. Теперь пеплоходцы теряют свой бег на станции. Увынск.
4. Шаман пеплоходцев теперь может чувствовать местонахождение мировой кузницы.

NightDawnFox committed:
Смешное добавление пеплоходцам

Теперь все магмитовые вещи могут использоваться пеплоходцами

NightDawnFox committed:
обновляем названия карты

NightDawnFox committed:
Починка атмоса

Никакого нормального давления на картах (возможно я всё сломал)

NightDawnFox commented:
Так и не разобрался, почему в коммиты залезли лишние файлы. Ну и ладно, главное, что вроде всё в норме..

aeternaclose commented:
Road to Contributor

NightDawnFox committed:
Update gems.dm
Revert "Revert "Update gems.dm""

This reverts commit cb3e95db92445e7e314fd6427cfe426047255926.
Revert "Update gems.dm"

This reverts commit 72d951ff6a28d1c35443f24d9835a1e1e8dfee1b.

NightDawnFox committed:
Update tgui.bundle.js

Да.

NightDawnFox committed:
фиксы проблем..

NightDawnFox committed:
фиксы ч.2

NightDawnFox committed:
ультра-балансная правка.

Den109G commented on code/game/objects/items/weapons/twohanded.dm:
```diff
@@ -1002,3 +1002,28 @@
+		icon_state = "fishing_rod0"
+	..()
+
+/obj/item/twohanded/fishingrod
```
А что оно делает в конце файла? Вроде не нужно.

NightDawnFox replied:
оно в конце что-то должно было делать, но я по итогу забил :clueless:
Поправлено

NightDawnFox committed:
хотфикс номер раз.

NightDawnFox committed:
Изменения по запросу администрации

Лампа сказал, что обидится, если я повышу цены на подрывные заряды, поэтому я сделал их дешевле..
Резонатор чутка подешевле по причине "никто не использует"
Переход резака в раздел "mining tools"

NightDawnFox committed:
ладно.

NightDawnFox committed:
Фиксы карты + фиксы ДМИ + фиксы тгуи

Dimach committed:
Fix dmi

Dimach committed:
screen fog
